app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/TrackListViewModel.kt

```diff
@@ -152,13 +160,18 @@ constructor(
     skip(-1)
   }
 
+  fun toggleLoop() {
+    _uiState.value = _uiState.value.copy(isLooping = !_uiState.value.isLooping)
```

Comment:
I think it would be better if you put some kind of way to have it turn to True or false instead of only switching from one.
You could overload the function or put a default argument `fun toggleLoop(val isLooping = !_uiState.value.isLooping) { ...`

Reply:
Will do, thanks for the suggestion